src/files.ts

```diff
@@ -112,16 +113,17 @@ export async function getProjectFiles(rootDir: string = path.join('.', '/'), cal
 
         // check ignore files
         const ignoreMatches = multimatch(filePaths, ignorePatterns, { dot: true });
+        const intersection: string[] = filePaths.filter(file => !ignoreMatches.includes(file));
```

Comment:
This gets the files that are not ignored. Before this, we were not really using `multimatch` that effectively.

Reply:
Put these GitHub comments in the source code (where appropriate).